kpi/deployment_backends/kobocat_backend.py

```diff
@@ -864,8 +864,9 @@ def bulk_update_submissions(
             kc_request = requests.Request(
                 method='POST', url=self.submission_url, files=files
             )
+            # `asset.owner` permissions needed for kobocat to allow the request
             kc_response = self.__kobocat_proxy_request(
-                kc_request, user=requesting_user
+                kc_request, user=self.asset.owner
```

Comment:
Why did you revert it? We still want `requesting_user` to make the request, right?

Reply:
@noliveleger I needed to revert it for bulk editing. I shouldn't have changed it to begin with.